src/__tests__/logic.test.js

```diff
@@ -0,0 +1,52 @@
+import calculate from '../logic/calculate';
+import operate from '../logic/operate';
+
+const obj = {
+  total: '450',
+  next: '50',
+  operation: '+',
+};
+
+describe('Test calculator logic', () => {
+  it('should add two values and give answer', () => {
+    const { total } = calculate(obj, '=');
+    expect(total).toStrictEqual('500');
+  });
+
+  it('should clear total', () => {
+    const { total } = calculate(obj, 'AC');
+    expect(total).toStrictEqual(null);
+  });
+
+  it('shoud return divide by zero', () => {
+    obj.total = '0';
+    obj.next = '0';
+    const { total } = calculate(obj, '%');
+    expect(total).toStrictEqual('0');
+  });
+});
```

Comment:
- [ ] Great job on writing the unit test for your project as specified in the requirement! 👏 However, Your calculate test doesn't cover all the calculating logic. Please make sure to write your test to cover all the logic in the `calculate.js` file.